From cb38b363ca4f35e2098a046b55a1285bab5aae17 Mon Sep 17 00:00:00 2001
From: Denys Vlasenko <dvlasenk@redhat.com>
Date: Tue, 18 Oct 2011 19:06:04 +0200
Subject: [PATCH 2/5] abrt_rh_support: mark a few strings for translation

Signed-off-by: Denys Vlasenko <dvlasenk@redhat.com>
---
 src/plugins/abrt_rh_support.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/plugins/abrt_rh_support.c b/src/plugins/abrt_rh_support.c
index aff34c6..05396e2 100644
--- a/src/plugins/abrt_rh_support.c
+++ b/src/plugins/abrt_rh_support.c
@@ -393,16 +393,16 @@ post_case_to_url(const char* url,
         errmsg = case_state->curl_error_msg;
         if (errmsg && errmsg[0])
         {
-            result->msg = xasprintf("error in case creation: %s", errmsg);
+            result->msg = xasprintf(_("error in case creation: %s"), errmsg);
         }
         else
         {
             errmsg = case_state->body;
             if (errmsg && errmsg[0])
-                result->msg = xasprintf("error in case creation, HTTP code: %d, server says: '%s'",
+                result->msg = xasprintf(_("error in case creation, HTTP code: %d, server says: '%s'"),
                         case_state->http_resp_code, errmsg);
             else
-                result->msg = xasprintf("error in case creation, HTTP code: %d",
+                result->msg = xasprintf(_("error in case creation, HTTP code: %d"),
                         case_state->http_resp_code);
         }
         result->body = case_state->body;
@@ -453,7 +453,7 @@ create_new_case(const char* base_url,
         /* Case Creation returned valid code, but no location */
         result->error = -1;
         free(result->msg);
-        result->msg = xasprintf("error in case creation: no Location URL, HTTP code: %d",
+        result->msg = xasprintf(_("error in case creation: no Location URL, HTTP code: %d"),
                 result->http_resp_code
         );
     }
-- 
1.7.6.4

